monGARS/core/cortex/curiosity_engine.py

```
@@ -6,7 +6,7 @@
 import math
 import types
 from collections.abc import AsyncIterator, Iterable, Mapping, Sequence
-from contextlib import asynccontextmanager
+from contextlib import asynccontextmanager, suppress
```

Flake8 (7.3.0)

[error] 'contextlib.suppress' imported but unused

(F401)